cns/service/main.go

```diff
@@ -785,6 +785,7 @@ func main() {
 				KeyVaultCertificateName:            cnsconfig.KeyVaultSettings.CertificateName,
 				MSIResourceID:                      cnsconfig.MSISettings.ResourceID,
 				KeyVaultCertificateRefreshInterval: time.Duration(cnsconfig.KeyVaultSettings.RefreshIntervalInHrs) * time.Hour,
+				UseMTLS:                            cnsconfig.UseMTLS,
```

Comment:
if tlssetting is set when UseHttps is true, then I am wondering how it is working for ACI. We need to follow up on this with ACI

Reply:
i think you already followed up on this with ACI - resolving